test/integration/requests/accountCurrencies.ts

```diff
@@ -0,0 +1,47 @@
+import { assert } from 'chai'
+import _ from 'lodash'
+
+import { AccountCurrenciesRequest } from 'xrpl-local'
+
+import serverUrl from '../serverUrl'
+import { setupClient, suiteClientSetup, teardownClient } from '../setup'
+
+// how long before each test case times out
+const TIMEOUT = 20000
+
+describe('AccountCurrencies', function () {
```

Comment:
```suggestion
describe('account_currencies', function () {
```
you get the idea, I'm gonna stop doing every single one of these